pkg/cmd/admin/policy/reconcile_sccs.go

```diff
@@ -0,0 +1,241 @@
+package policy
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kapierrors "k8s.io/kubernetes/pkg/api/errors"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+
+	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	"k8s.io/kubernetes/pkg/util/sets"
+)
+
+// ReconcileSCCRecommendedName is the recommended command name
+const ReconcileSCCRecommendedName = "reconcile-sccs"
+
+type ReconcileSCCOptions struct {
+	Confirmed bool
```

Comment:
Yep, understood.  The problem is here https://github.com/pweil-/origin/blob/6f86f784806558c188170ac235a994d3482c02f5/pkg/cmd/server/origin/ensure.go#L165-165.  I can't just use what is in the default SCCs from bootstrap because we manipulate it with settings from the MasterConfig when installing defaults.  I'll look for a better way to do it